sale_stock_picking_validation_blocking/views/sale_order.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="sale_order_form_view">
+        <field
+            name="name"
+        >sale.order.form (in sale_stock_picking_validation_blocking)</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_stock.view_order_form_inherit_sale_stock" />
+        <field name="arch" type="xml">
+            <button name="action_draft" position="after">
+                <field name="picking_validation_blocked" invisible="1" />
```

Comment:
Maybe it's better to show this field somewhere on the form. I thought the blocked state was not visually very obvious with the buttons below. Besides these buttons are not visible to people without the sales manager group.

Reply:
Our customer suggested to track this field, would that do the job for you ?
Thanks for the review.

Reply:
That could do, but the track message might be buried deep down in case there is a lot of communication on the sales order. Another idea is to add a smart button that's visible to everyone, and show a warning message when someone with no access right clicks on it, if that makes sense.

Reply:
I've pushed your suggestions.
I didn't quite liked to have a button which you can't click on, but the field is now displayed in the 'Delivery' section (+ tracked).